contracts/src/BeefyClient.sol

```diff
@@ -601,6 +601,14 @@ contract BeefyClient {
         return numValidators - (numValidators - 1) / 3;
     }
 
+    /**
+     * @dev We have 2/3rd +1 honesty assumption on polkadot validators. Hence it is sufficient (for both random sampling and Fiat Shamir) to check 1/3rd +1 validator signatures to ensure at least 1 honest validator signed the payload.
+     * @param numValidators The number of validators in the validator set.
+     */
+    function computeCheckQuorum(uint256 numValidators) internal pure returns (uint256) {
```

Comment:
Strictly speaking, in terms of meaning, this function is not computing a quorum, but the maximum number of required signatures for both algorithms. So would the following be a better name?

```suggestion
    function computeMaxRequiredSignatures(uint256 numValidators) internal pure returns (uint256) {
```

Reply:
https://github.com/Snowfork/snowbridge/pull/1659/commits/a17220c7c14f8c35893b53da5a9e6e249a55509f